docs/user-guide.md

```diff
@@ -24,26 +24,26 @@ that might be null then you write `@Nullable String x;`.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, references have one of three possible properties
```

Comment:
Follow-up to https://github.com/jspecify/jspecify/pull/201#discussion_r663097830

Maybe not saying anything works well enough. The other case we're concerned about is "parametric nullness", which we'll discuss later with generics/wildcards. But one could see that as another case where we don't know whether it can be null or not (if the bound is nullable) or that they can't be null (if the bound is non-null).
Or am I missing what other case "most" was referring to?

Reply:
I actually don't remember why I wrote "most" here and I think it can reasonably be omitted.